### DISCUSSION TITLE
Compilation using GCC-12.X

### DISCUSSION THREAD

**Discussion by GuardZ** [mdolab/dafoam#570] [General]
# Description of feature
Hi~, sorry to bother you. Would you guys have the plan to compile the dafoam with GCC-12.X?

After the check of OpenFOAM-v1812, I found that it only supports to be compiled with no more than GCC-9.X.

# Potential solution
Modify the configure setting file ( compiler flags, information about GCC-12.X, etc) in OpenFOAM-v1812?

**Comment by friedenhe** [comment]
This could be useful for newer OS. However, compiling it with Gcc 12. x is not our priority. I suggest you make Gcc using ThirdParty-v1812/makeGcc. Then, use the ThirdParty/Gcc to compile OF.
